docs/content/installation/install_compatibility_requirements.md

```diff
@@ -50,7 +50,7 @@ Complete these steps to prepare your environment for installing the CSI (Contain
 
 2. Configure your storage system host attachment, per worker node.
 
-    **Note:** IBM® block storage CSI driver 1.11.0 introduces dynamic host definition. For more information and installation instructions, see [Installing the host definer](install_hostdefiner.md). If this feature is not installed, the nodes are not dynamically defined on the storage system and they must be defined manually.
+    **Note:** IBM® block storage CSI driver 1.11.1 introduces dynamic host definition. For more information and installation instructions, see [Installing the host definer](install_hostdefiner.md). If this feature is not installed, the nodes are not dynamically defined on the storage system and they must be defined manually.
```

Comment:
1.11.1 does not introduce the host definition feature, is that by mistake or we must refer to the latest version in this line?
since 1.11.0 didn't as well, 1.10.0 did as I remember.